openslides/core/static/templates/index.html

```diff
@@ -9,6 +9,7 @@
 <link rel="stylesheet" href="static/css/openslides-libs.css">
 <link rel="stylesheet" href="static/css/app.css">
 <script src="static/js/openslides-libs.js"></script>
+<script src="static/ckeditor/ckeditor.js"></script>
```

Comment:
It is an unknown reason. Please try to add
```json
"overrides": {
  "ckeditor": {
    "main": [
      "ckeditor.js"
    ]
  }
}
```
to bower.json. Then remove the line 12 in the index.html, run gulp and the ckeditor.js is now included our libs. But for unknown reason CKEditor now tries to load more files like icon images and language files from the wrong path instead of the global static path ... and crashes.